homeassistant/components/whirlpool/quality_scale.yaml

```diff
@@ -0,0 +1,26 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      The integration does not provide any additional actions.
+  appropriate-polling: done
+  brands: done
```

Comment:
Let's evaluate all rules

Reply:
You mean the other levels? I plan on adding them progressively, to keep the PRs smaller and easier/quicker to review.

Adding everything at once goes a bit against point 1 of https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr :smiley:
We should have a rule of "one level per PR" like we do for platforms :)

Reply:
Added the remaining rules.